src/fidesops/main.py

```diff
@@ -47,6 +51,21 @@ def start_webserver() -> None:
     if config.database.ENABLED:
         logger.info("Starting scheduled request intake...")
         initiate_scheduled_request_intake()
+    import os
+
+    if os.getenv("FIDESOPS__ROOT_USER__ANALYTICS_ID"):
```

Comment:
Oh look you're using `FIDESOPS__ROOT_USER__ANALYTICS_ID`! I think you do need to explicitly define the prefix in `config.py` though, no?